ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/S3IntegrationForm.tsx

```diff
@@ -328,21 +335,23 @@ function S3IntegrationForm({
                             isDisabled={!isEditable}
                         />
                     </FormLabelGroup>
-                    <FormLabelGroup
-                        label=""
-                        fieldId="externalBackup.s3.useIam"
-                        touched={touched}
-                        errors={errors}
-                    >
-                        <Checkbox
-                            label="Use container IAM role"
-                            id="externalBackup.s3.useIam"
-                            isChecked={values.externalBackup.s3.useIam}
-                            onChange={updateKeysOnChange}
-                            onBlur={handleBlur}
-                            isDisabled={!isEditable}
-                        />
-                    </FormLabelGroup>
+                    {canUseCloudBackupIntegrations && (
+                        <FormLabelGroup
```

Comment:
The thing is that @connorgorman suggests to entirely remove Cloud Backup integrations, both S3 and GCS. See https://redhat-internal.slack.com/archives/C055ZNPE8F8/p1684251678887899

Not just hide the IAM option from S3 backup.

![image](https://github.com/stackrox/stackrox/assets/537715/aea276ac-fcc4-4c39-93d5-adf5f43c361a)